core/lib/mno_enterprise/concerns/models/ability.rb

```diff
@@ -162,10 +168,24 @@ def impac_abilities(user)
   end
 
   # Abilities for admin user
-  def admin_abilities(user)
-    if user.admin_role.to_s.casecmp('admin').zero? || user.admin_role.to_s.casecmp('staff').zero?
+  def admin_abilities(user, session)
+    # TODO: Implement full scale staff role abilities.
+    if user.admin? || user.staff?
       can :manage_app_instances, MnoEnterprise::Organization
       can :manage_sub_tenant, MnoEnterprise::SubTenant
+    elsif user.support?
+      # Support users are 'logged in' when they have the support_org_id stored inside their session.
+      can :read, MnoEnterprise::Invoice do |invoice|
```

Comment:
Note we don't need these accesses for admins/staff because the `authorize!` methods will only be called if the user is support.  see https://github.com/maestrano/mno-enterprise/pull/751/files#diff-cb1a9bf8b5b2cd03133510e320d9c813R24

Once we have a full scale admin and staff abilities system set up, we can remove these conditionals.